samples/spring-boot/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openjdk:11
```

Comment:
Wouldn't be better to build with jib?

Reply:
I would perfer jib, it is much nicer solution imo.

Reply:
Also it is used on other projects.

Reply:
Changed.